modules/ROOT/pages/customize.adoc

```diff
@@ -1,101 +1,177 @@
-= Customize the Look and Feel of Your API Community
+= API Community Look and Feel Customization
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Use the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] to customize the branding, look and feel, and site structure of your community pages to match your desired brand identity.
+The https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] enables
```

Comment:
```suggestion
https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] enables
```
Never precede a proper noun with an article, unless the proper noun is followed by a common noun object.
GLOBALLY standardize references to Community Builder; avoid just Builder.